ivy/functional/backends/torch/statistical.py

```diff
@@ -14,23 +14,39 @@
 # -------------------#
 
 
-@with_unsupported_dtypes({"2.1.0 and below": ("complex",)}, backend_version)
+@with_unsupported_dtypes({"2.1.0 and below": ("complex", "bool")}, backend_version)
```

Comment:
Same here, `torch.min` should ideally be working with `bool`, could you please test that as well?